### ISSUE TITLE
Update deps to protractor ^5.0.0

### ISSUE THREAD

**Issue by wyvern8** [ReadyTalk/cukefarm#42]
looks like we need to update dependencies to new major version.  seems ok locally

**Comment by thompsnm** [comment]
Thanks for the heads up, I'll try to upgrade and test today.

Sorry for the belated response on this one.

**Comment by wyvern8** [comment]
No probs - i actually ended up creating a fork with this and a couple of
other minor tweaks - it is just publishing back to GitHub release tag atm.
I have not put the waiting for element in yet, but am using it locally via
Expected conditions which seems to help with slower test agents.

**Comment by thompsnm** [comment]
Just to give an update on this: our standard procedure is to update all dependencies when we're preparing a new major version. Since the cucumber-js project has been [publishing release candidates since November](https://github.com/cucumber/cucumber-js/releases) I think it is prudent to upgrade to Cucumber v2 with this release of CukeFarm. I spent Friday familiarizing myself with what's new in Cucumber v2, and it will require some rewrite of of the GeneralStepDefs file. Given the scope of what _needs_ to change, now seems like a good time to tackle some of the long running issues and address some tech debt in the project. I plan to make more progress on that today; expect more details later this week.

**Comment by thompsnm** [comment]
It's been a while since I've looked at this project, but from what I can tell the `v5.0.0` version has been released and has support for Protractor `v5.0.0`:

```
$ npm show cukefarm
{ name: 'cukefarm',
  description: 'An opinionated template for writing Cucumber tests with Protractor',
  'dist-tags': { latest: '5.0.0' },
  versions:
   [ '1.0.0',
     '1.0.1',
     '2.0.0',
     '2.0.1',
     '2.0.2',
     '3.0.0',
     '3.0.1',
     '4.0.0',
     '5.0.0-rc.1',
     '5.0.0-rc.2',
     '5.0.0' ],
  maintainers: [ 'thompsnm <nathan.mike.thompson@gmail.com>' ],
  author: 'Nathan Thompson',
  time:
   { '2.0.0': '2015-08-13T01:19:22.618Z',
     '2.0.1': '2015-11-03T22:48:54.113Z',
     created: '2015-03-10T18:00:50.436Z',
     '5.0.0-rc.2': '2017-05-12T19:23:52.097Z',
     '5.0.0-rc.1': '2017-03-13T22:21:01.874Z',
     '4.0.0': '2016-08-14T03:30:37.982Z',
     '2.0.2': '2015-12-01T22:08:29.233Z',
     '3.0.0': '2015-12-07T22:45:41.624Z',
     modified: '2017-08-11T21:04:43.486Z',
     '5.0.0': '2017-08-11T21:04:43.486Z',
     '3.0.1': '2016-01-20T15:43:26.153Z',
     '1.0.0': '2015-03-10T18:00:50.436Z',
     '1.0.1': '2015-04-13T14:30:47.001Z' },
  repository:
   { type: 'git',
     url: 'git+https://github.com/ReadyTalk/cukefarm.git' },
  users: {},
  readmeFilename: 'README.md',
  homepage: 'https://github.com/ReadyTalk/cukefarm',
  keywords: [ 'cucumber', 'cucumberjs', 'protractor', 'selenium' ],
  bugs: { url: 'https://github.com/ReadyTalk/cukefarm/issues' },
  license: 'MIT',
  version: '5.0.0',
  engines: { node: '>= 6.10.0', npm: '>= 3.10.10' },
  main: 'index.js',
  dist:
   { tarball: 'https://artifactory.pgi-tools.com/artifactory/api/npm/npm/cukefarm/-/cukefarm-5.0.0.tgz',
     shasum: '323d1582c78e5efb913ae9a29a278c863f0c2f91' },
  dependencies: { chai: '^3.5.0', 'chai-as-promised': '^5.3.0', q: '^1.4.1' },
  devDependencies:
   { cucumber: '^2.0.0',
     docha: 'git://github.com/thompsnm/docha.git#09b60cd',
     express: '^4.14.1',
     'js-fixtures': '^1.5.3',
     mocha: '^3.2.0',
     protractor: '^5.0.0',
     'protractor-cucumber-framework': '^3.0.0',
     sinon: '^1.17.7',
     'sinon-as-promised': '^4.0.2',
     'sinon-chai': '^2.8.0' },
  peerDependencies:
   { 'protractor-cucumber-framework': '^3.0.0',
     cucumber: '^2.0.0',
     protractor: '^5.0.0' },
  directories: {},
  scripts:
   { ci: 'npm run test && npm run generateApiDocs',
     generateApiDocs: 'node util/generateApiDocs.js',
     test: 'node util/runProtractorTests.js',
     webdriverUpdate: 'webdriver-manager update' },
  gitHead: '879902424d629bb1a8ee7289ee49ac67ba94afb0' }
```
As such, I am closing this issue.
